utils.c

```diff
@@ -108,6 +147,12 @@ pymod_get_string(PyObject* obj, TRIE_LETTER_TYPE** word, ssize_t* wordlen) {
 #endif
 }
 
+#if defined PEP393_UNICODE
+#    define pymod_get_string_(obj, word, wordlen, is_copy) pymod_get_string(obj, word, wordlen, is_copy)
```

Comment:
I suggest to simply extend `pymod_get_string` signature with is_copy flag and get rid of this macro. The worst we might get is compilation warning about an unused parameter.